src/hero.css.ts

```diff
@@ -0,0 +1,66 @@
+import {createVar, style, fallbackVar} from '@vanilla-extract/css';
+import * as mq from './media-queries.css';
+import {sprinkles} from './sprinkles.css';
+import {
+    LARGE_DESKTOP_MAX_WIDTH,
+    MOBILE_SIDE_MARGIN,
+    SMALL_DESKTOP_SIDE_MARGIN,
+    TABLET_SIDE_MARGIN,
+} from './responsive-layout.css';
+
+const height = createVar();
+
+export const vars = {height};
+
+export const container = style([
+    sprinkles({display: 'flex'}),
+    {
+        minHeight: 400,
+        height: fallbackVar(height, '100%'),
+        '@media': {
+            [mq.desktopOrBigger]: {
+                minHeight: 460,
+                height: fallbackVar(height, '100%'),
+            },
+        },
+    },
+]);
+
+export const containerMobile = sprinkles({flexDirection: 'column'});
+
+export const containerDesktop = sprinkles({alignItems: 'center', height: '100%'});
+
+export const actions = style([
+    sprinkles({display: 'flex', flex: 1, alignItems: 'flex-end'}),
+    {marginTop: 24},
+]);
+
+export const layout = style([
```

Comment:
I'd include a comment here explaining why we needed to duplicate this styles from ResponsiveLayout